deploy/import-upstream.sh

```diff
@@ -1,2 +1,14 @@
-oc import-image coreos/prometheus-operator:v0.25.0 --from=quay.io/coreos/prometheus-operator:v0.25.0 --confirm
-oc import-image coreos/configmap-reload:v0.0.1 --from=quay.io/coreos/configmap-reload:v0.0.1 --confirm
+#!/bin/sh
+oc import-image prometheus:v2.11.104 --from=quay.io/prometheus/prometheus --confirm
```

Comment:
Let's pick a specific tag rather than relying on `latest`. Comment goes for all the components. We'll need to do the same thing for the downstream script as well.

Reply:
Should we tag everything uniformly, including latest and specific versions?

Reply:
At this point I think I've resolved this.